src/Azure.Functions.Cli/Helpers/GlobalCoreToolsSettings.cs

```diff
@@ -19,15 +19,40 @@ public static WorkerRuntime CurrentWorkerRuntime
             }
         }
 
+        public static WorkerRuntime CurrentWorkerRuntimeOrNone
+        {
+            get
+            {
+                return _currentWorkerRuntime;
+            }
+        }
+
+        public static string CurrentLanguageOrNull { get; private set; } = null;
+
         public static void Init(ISecretsManager secretsManager, string[] args)
         {
             try
             {
-                if (args.Contains("--csharp") || args.Contains("--dotnet"))
```

Comment:
@ahmelsayed Did you missed F#?

Reply:
I don't think we have F# templates for the core-tools.